cfme/tests/control/test_default_alerts.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+import os
+import pytest
+import yaml
+
+from cfme.utils.path import data_path
+
+@pytest.fixture
+def default_alerts(appliance):
+    file_name = data_path.join('ui/control/default_alerts.yaml').strpath
+    alerts = {}
+    if os.path.exists(file_name):
```

Comment:
Need an else here in case the file DNE for whatever reason, where you call pytest.skip(). Otherwise we'll return an empty list.